packages/help-center/src/components/success-icon.tsx

```diff
@@ -1,22 +1,7 @@
 export const SuccessIcon: React.FC = () => (
-	<svg width="70" height="70" viewBox="0 0 70 70" fill="none" xmlns="http://www.w3.org/2000/svg">
-		<path
-			d="M68.6749 34.9921C68.6749 43.7173 65.2099 52.0854 59.0417 58.2565C52.8736 64.4276 44.5072 67.8966 35.782 67.9008C29.2051 58.6978 25.5391 47.734 25.2577 36.426C24.9764 25.118 28.0925 13.9855 34.2036 4.46683L35.782 2.0835C44.5072 2.08768 52.8736 5.55667 59.0417 11.7278C65.2099 17.8989 68.6749 26.2669 68.6749 34.9921Z"
-			fill="#EFEFEF"
-		/>
-		<path
-			d="M68.6745 34.9922C68.6745 43.9211 65.1275 52.4843 58.8139 58.798C52.5002 65.1116 43.9371 68.6586 35.0082 68.6586C26.0793 68.6586 17.5162 65.1116 11.2025 58.798C4.88884 52.4843 1.34188 43.9211 1.34188 34.9922C1.33725 32.9271 1.52752 30.866 1.91011 28.8366C3.44563 20.5851 8.00795 13.2047 14.7026 8.14248C21.3972 3.08021 29.7414 0.701285 38.0991 1.47185C46.4568 2.24242 54.2252 6.10701 59.8814 12.308C65.5376 18.509 68.6736 26.5991 68.6745 34.9922V34.9922Z"
-			stroke="#003A5F"
-			strokeWidth="1.68"
-			strokeLinecap="round"
-			strokeLinejoin="round"
-		/>
-		<path
-			d="M23.334 37.9118L31.5091 45.0617L47.1664 23.675"
-			stroke="#003A5F"
-			strokeWidth="3"
-			strokeLinecap="round"
-			strokeLinejoin="round"
-		/>
+	<svg width="70px" height="70px" viewBox="0 0 2.8 2.8" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g id="icomoon-ignore"/>
```

Comment:
```suggestion
	<svg width="70px" height="70px" viewBox="0 0 2.8 2.8" version="1.1" xmlns="http://www.w3.org/2000/svg">
```

Looks like the `<g id="icomoon-ignore"/>` element is not used or referenced, so let's remove it altogether.
`xmlns:xlink="http://www.w3.org/1999/xlink"` attribute is causing errors with React:
```
SyntaxError: /home/runner/work/wp-calypso/wp-calypso/packages/help-center/src/components/success-icon.tsx: Namespace tags are not supported by default. React's JSX doesn't support namespace tags.
```
([see the test failures for this PR](https://github.com/Automattic/wp-calypso/actions/runs/8100218117/job/22137700483))
This is because while React's JSX/TSX syntax looks like HTML, it's actually a bit modified version of it. For example, you don't specify a CSS class name on an element using the `class` attribute, but instead `className`. And, as the error mentions, `xlmns:xlink` attribute is not supported either (just `xlmns` is fine, though, so we can keep that).